src/megameklab/com/ui/MegaMekLabMainUI.java

```diff
@@ -44,36 +50,76 @@
     private static final long serialVersionUID = 3971760390511127766L;
 
     private Entity entity = null;
-
+    protected JPanel masterPanel = new JPanel();
+    protected JScrollPane scroll = new JScrollPane();
+    protected MenuBarCreator menubarcreator;
+    
     public MegaMekLabMainUI() {
 
         EquipmentType.initializeTypes();
         MechSummaryCache.getInstance();
         new CConfig();
         UnitUtil.loadFonts();
-        System.out.println("Starting MegaMekLab version: " + MegaMekLab.VERSION);
-        
+        System.out.println("Starting MegaMekLab version: " + MegaMekLab.VERSION);        
         setLookAndFeel();
```

Comment:
This doesn't get called until after the main splash screen, which means that when you click the button to load the unit, the unit selector dialog can have a different look and feel than the the rest of the ui. I don't think it really belongs here in the first place, since it doesn't need to run every time the unit type changes, but now it needs to move to somewhere before the startup splash screen.

Reply:
That should probably in the startup method of MegaMekLab.java. The System.out.println should also be there I think.

Reply:
I moved `setLookandFeel()` and 'system.out.println` to MegaMekLab.startup()